pyo3-macros-backend/src/attributes.rs

```diff
@@ -45,6 +45,7 @@ pub mod kw {
     syn::custom_keyword!(unsendable);
     syn::custom_keyword!(weakref);
     syn::custom_keyword!(gil_used);
+    syn::custom_keyword!(default);
```

Comment:
I think this might not be necessary, I think `syn::Token![default]` works.

https://docs.rs/syn/latest/syn/token/struct.Default.html